packages/flutter_test/lib/src/all_elements.dart

```diff
@@ -20,6 +20,15 @@ Iterable<Element> collectAllElementsFrom(
   Element rootElement, {
   @required bool skipOffstage,
 }) {
+  assert(() {
+    if (rootElement == null) {
+      throw FlutterError(
+        'A root widget has not been attached to the render tree. '
+        'Have you remembered to call runApp()?'
```

Comment:
Is `runApp` really the right hint here? This is called from within `flutter_test` and (usually) you don't call runApp directly from your test.